build-system/server/new-server/transforms/transform.ts

```diff
@@ -21,12 +21,16 @@ import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
 import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
+//import transformSxg from './sxg/sxg-transform';
 
 const argv = minimist(process.argv.slice(2));
 const transforms = [transformStories(), transformScriptPaths(), transformCss()];
 
 export async function transform(fileLocation: string): Promise<string> {
-  if (argv.esm) {
+  if (argv.sxg){
+    //transforms.unshift(transformSxg());
```

Comment:
?

Reply:
just clean this up i think or comment out the whole block (including the if statement) and and add a `TODO` comment.